docs/hooks/hook_civicrm_navigationMenu.md

```diff
@@ -134,7 +134,8 @@ function donortrends_civicrm_navigationMenu(&$params) {
       'separator' => 1,
       'parentID' => $reportID,
       'navID' => $navId,
-      'active' => 1
+      'active' => 1,
+      'attr' => ['target' => '_blank'],   // if you want to open this menu link in a new page, then add '_target' attribute
```

Comment:
```suggestion
      'attr' => ['target' => '_blank'],
```
We try and avoid code comments to document implementation. Can you document the "attr" parameter in the Parameters section of the document?